.clang-format

```diff
@@ -165,6 +165,9 @@ StatementMacros:
   - TOML_ATTR
   - TOML_CONST_GETTER
   - TOML_CONST_INLINE_GETTER
+  - TOML_EXPORTED_CLASS
+  - TOML_EXPORTED_MEMBER_FUNCTION
+  - TOML_EXPORTED_FREE_FUNCTION
```

Comment:
Ah, looks like I forgot `TOML_EXPORTED_STATIC_FUNCTION` here. Mind adding it for me?